src/App.js

```diff
@@ -2,24 +2,27 @@ import React from 'react';
 
 import './App.scss';
 
-const App = () => {
-  setInterval(() => {
-    const date = new Date();
+class App extends React.Component {
+  state = {};
```

Comment:
provide initial value for date(now it's empty page on first second)

Reply:
+